dojo/tools/ossindex_devaudit/parser.py

```diff
@@ -0,0 +1,100 @@
+import json
+
+from dojo.models import Finding
+
+
+class OssIndexDevauditParser(object):
+    def __init__(self, json_file, test):

```

Comment:
Could you add a comment with some details about the tool or references describing where the data come from?

Reply:
just add a comment to the class. you could take exemple from [https://defectdojo.readthedocs.io/en/latest/integrations.html](https://defectdojo.readthedocs.io/en/latest/integrations.html)

Reply:
Comment added